platform/semux/api.go

```diff
@@ -0,0 +1,92 @@
+package semux
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/models"
+	"github.com/trustwallet/blockatlas/util"
+	"net/http"
+	"strconv"
+)
+
+var client *Client
+
+// Setup registers the semux route
+func Setup(router gin.IRouter) {
+	router.Use(util.RequireConfig("semux.api"))
+	router.Use(withClient)
+	router.GET("/:address", getTransactions)
+}
+
+func getTransactions(c *gin.Context) {
+	s, err := client.GetTxsOfAddress(c.Param("address"))
+	if apiError(c, err) {
+		return
+	}
+
+	txs := make([]models.Tx, len(s))
```

Comment:
Your `Normalize` can return empty transactions/errors. Please ignore errored transactions instead of including them.
Something like this:

```go
var txs []models.Tx
for i, srcTx := range s {
    tx, ok := Normalize(&srcTx)
    if ok {
        txs = append(txs, tx)
...
```

Reply:
fixed in 1ecddbe99a1616a199d864847b10cd642d50549f